### PR TITLE
docs: add faq section

### PR DESCRIPTION
**Pull request by Nr18**
**Issue #, if available:**

## Description of changes:

- add instructions on how you could enable `botocore` and `boto3` logs

**Checklist**

 

* [x] [Meet tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
* [x] Update tests
* [x] Update docs
* [x] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-python/blob/376ec0a2ac0d2a40e0af5717bef42ff84ca0d1b9/.github/semantic.yml#L2)

## Breaking change checklist

**RFC issue #**:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/core/logger.mdx [hunk 1]
```diff
@@ -406,3 +406,23 @@ def test_lambda_handler(lambda_handler, lambda_context):
     test_event = {'test': 'event'}
     lambda_handler(test_event, lambda_context) # this will now have a Context object populated
 ```
+
+## FAQ
+
+**Why don't I see my library logs?**
```
Comment (heitorlessa) [inline docs/content/core/logger.mdx:412]: That's not true anymore since 1.7.0, or 9 days ago to be more precise :)

File: docs/content/core/logger.mdx [hunk 2]
```diff
@@ -406,3 +406,23 @@ def test_lambda_handler(lambda_handler, lambda_context):
...
+
+Because Lambda configures the root logger for you and using the `Logger` utility would then result in duplicate log lines
+it removes the handler from the root logger.
+
+**How can I enable boto3 and botocore library logging?**
+
+You can enable the `botocore` and `boto3` logs by adding the following:
```
Comment (heitorlessa) [inline docs/content/core/logger.mdx:419]: Suggestion: add some context as to what this function from boto does, since not everyone might know ;)

TL;DR: 'set_stream_logger' configures boto package logger with a Log Handler (stdout) and log level as DEBUG.

**Review by heitorlessa — CHANGES_REQUESTED** [review]
Thanks a lot for writing this out ;)

One minor improvement only

File: docs/content/core/logger.mdx (hunk 1 above)
Comment (Nr18) [reply docs/content/core/logger.mdx]: Ah right... I totally missed that :-) Seen it working now that I actually tried enabling the `boto3` and `botocore` logs myself ;-)

File: docs/content/core/logger.mdx (hunk 2 above)
Comment (Nr18) [reply docs/content/core/logger.mdx]: Makes sense I added the context!

**Review by heitorlessa — APPROVED** [review]
Looks great! Thanks a lot for this @Nr18 !!
